doc/old/getting_started.rst

```diff
@@ -18,14 +18,22 @@ and radiotherapy objects.
 structures for easy manipulation. Modified datasets can be written again to
 DICOM format files.
 
-Here is a simple example of using *pydicom* in an interactive session, to read a
-radiotherapy plan file, change the patient setup from head-first-supine to
-head-first-prone, and save to a new file::
+Below is a simple example of using *pydicom* in an interactive session. In it we
+use the :func:`~pydicom.data.get_testdata_file` helper function to get the
+path to one of the pydicom test datasets, which in this case is a radiotherapy
+plan file. We then read the dataset from the path using :func:`~pydicom.filereader.dcmread`,
+which returns a :class:`~pydicom.dataset.FileDataset` instance `ds`. This is then
+used to print the *Patient Name* and change the *Patient Position* from
+head-first-supine (HFS) to head-first-prone (HFP). The changes are then saved to a
```

Comment:
Maybe remove or replace one or two of the "then"s...

Reply:
Done